src/parser/parse_tree_cl.c

```diff
@@ -743,6 +744,48 @@ pt_find_id_node (PARSER_CONTEXT * parser, PT_NODE * tree, void *void_arg, int *c
   return tree;
 }
 
+/*
+ * copy_node_in_term_dblink () - copies exactly a term node excluding cast node for dblink,
+     and returns a pointer to the copy. It is eligible for a walk "pre" function
+ *   return:
+ *   parser(in):
+ *   old_node(in):
+ *   arg(in):
+ *   continue_walk(in):
+ */
+static PT_NODE *
+copy_node_in_term_dblink (PARSER_CONTEXT * parser, PT_NODE * old_node, void *arg, int *continue_walk)
```

Comment:
suggestion of method name: pt_copy_term_for_dblink

the name like `copy_node_...` functions are for copying all kinds of nodes of tree structures.

Reply:
Ok, good point. thanks